docs/source/specifications/projjson.rst

```diff
@@ -71,19 +119,123 @@ The high level objects are:
   - ``Datum``
   - ``DatumEnsemble``
 
+.. _identifiers:
+
+Identifiers
++++++++++++
+
+All above mentioned objects can have an option ``id`` or ``ids`` member.
+
+The value of ``id`` is a JSON object with the following members:
+
+- ``authority``: (required) value of type string. e.g "EPSG", "OGC", "IGNF", etc.
+- ``code``: (required) value of type string or integer. e.g 4326 or "CRS84"
+- ``authority_citation``: (optional) value of type string that may be used to give
+  further details of the authority.
+- ``uri``: (optional) value of type string that may be used to give reference
+  to an online resource.
+
+An object can sometimes be identified in different ways, in which case the ``ids``
+member can be used to specify a JSON array of objects with the same type of ``id``.
+
+Identifiers are allowed in top-level objects and inner objects. The WKT2:2019
+specification recommends that if an object has an identifier, its inner objects
+should omit their identifiers, with the exceptions mentioned at
+http://docs.opengeospatial.org/is/18-010r7/18-010r7.html#37.
+
+Object usages
++++++++++++++
+
+CRS and coordinate operation objects are derived classes (in object modeling terminology)
+of a "object usage" class. An object usage has the following optional members:
+
+- ``scope``: (optional) value of type string describing the purpose or purposes of
+  the object. e.g "Geodesy, topographic mapping and cadastre"
+- ``area``: (optional) value of type string which describes a geographic area
+  over which a CRS or coordinate operation is applicable. e.g. "World"
+- ``bbox``: (optional) value of type object, with 4 required members:
+  * ``east_longitude``: (required) number expressing the longitude in degrees in [-180,180] range of the eastern most part of the extent.
+  * ``west_longitude``: (required) number expressing the longitude in degrees in [-180,180] range of the western most part of the extent. For an extent crossing the anti-meridian, west_longitude is lower than east_longitude.
+  * ``south_latitude``: (required) number expressing the latitude in degrees in [-90,90] range of the southern most part of the extent.
+  * ``north_latitude``: (required) number expressing the latitude in degrees in [-90,90] range of the northern most part of the extent.
+  The coordinates are expressed in a unspecified datum, with the longitudes
```

Comment:
If the CRS is a geographic one that specifies datum, would it be helpful / appropriate to encourage using that datum, but that otherwise it is unspecified (e.g., other planar CRS)?

Reply:
> If the CRS is a geographic one that specifies datum, would it be helpful / appropriate to encourage using that datum, but that otherwise it is unspecified (e.g., other planar CRS)?

we don't want to modify the semantic of the WKT CRS spec, which basically says unspecified datum, because given the intended accuracy of the bounding box of an area of use, an inaccuracy of ~ 100 meters don't matter